commit 7915bd3a855159b41e806234519bc89a1bee35d4
Author: Christopher Friedt <chrisfriedt@gmail.com>
Date:   Sun May 1 18:23:36 2011 -0400

    changed netinet/ether.h to netinet/if_ether.h and net/ethernet.h to net/if_ether.hj

diff --git a/networking/arp.c b/networking/arp.c
index 0ef267a..77e0a78 100644
--- a/networking/arp.c
+++ b/networking/arp.c
@@ -19,7 +19,7 @@
 #include <arpa/inet.h>
 #include <net/if.h>
 #include <net/if_arp.h>
-#include <netinet/ether.h>
+#include <netinet/if_ether.h>
 #include <netpacket/packet.h>
 
 #define DEBUG 0
diff --git a/networking/arping.c b/networking/arping.c
index 6f6b59c..ec81a0d 100644
--- a/networking/arping.c
+++ b/networking/arping.c
@@ -8,7 +8,7 @@
 
 #include <arpa/inet.h>
 #include <net/if.h>
-#include <netinet/ether.h>
+#include <netinet/if_ether.h>
 #include <netpacket/packet.h>
 
 #include "libbb.h"
diff --git a/networking/ether-wake.c b/networking/ether-wake.c
index 8f1479c..f440da3 100644
--- a/networking/ether-wake.c
+++ b/networking/ether-wake.c
@@ -66,8 +66,8 @@
 
 
 #include <netpacket/packet.h>
-#include <net/ethernet.h>
-#include <netinet/ether.h>
+#include <net/if_ether.h>
+#include <netinet/if_ether.h>
 #include <linux/if.h>
 
 #include "libbb.h"
diff --git a/networking/ifconfig.c b/networking/ifconfig.c
index da2635c..518d793 100644
--- a/networking/ifconfig.c
+++ b/networking/ifconfig.c
@@ -31,7 +31,7 @@
 #include <netinet/in.h>
 #if defined(__GLIBC__) && __GLIBC__ >=2 && __GLIBC_MINOR__ >= 1
 #include <netpacket/packet.h>
-#include <net/ethernet.h>
+#include <net/if_ether.h>
 #else
 #include <sys/types.h>
 #include <netinet/if_ether.h>
diff --git a/networking/ifplugd.c b/networking/ifplugd.c
index 58f56db..0567b7f 100644
--- a/networking/ifplugd.c
+++ b/networking/ifplugd.c
@@ -12,7 +12,7 @@
 #include <linux/if.h>
 #include <linux/mii.h>
 #include <linux/ethtool.h>
-#include <net/ethernet.h>
+#include <net/if_ether.h>
 #include <linux/netlink.h>
 #include <linux/rtnetlink.h>
 #include <linux/sockios.h>
diff --git a/networking/interface.c b/networking/interface.c
index 83af62b..167bb8d 100644
--- a/networking/interface.c
+++ b/networking/interface.c
@@ -33,7 +33,7 @@
 #include <net/if.h>
 #include <net/if_arp.h>
 #if (defined(__GLIBC__) && __GLIBC__ >= 2 && __GLIBC_MINOR__ >= 1) || defined(_NEWLIB_VERSION)
-# include <net/ethernet.h>
+# include <net/if_ether.h>
 #else
 # include <linux/if_ether.h>
 #endif
diff --git a/networking/libiproute/iplink.c b/networking/libiproute/iplink.c
index 82ab979..4d399b3 100644
--- a/networking/libiproute/iplink.c
+++ b/networking/libiproute/iplink.c
@@ -7,7 +7,7 @@
 #include <net/if.h>
 #include <net/if_packet.h>
 #include <netpacket/packet.h>
-#include <net/ethernet.h>
+#include <net/if_ether.h>
 
 #include "ip_common.h"  /* #include "libbb.h" is inside */
 #include "rt_names.h"
diff --git a/networking/libiproute/ll_proto.c b/networking/libiproute/ll_proto.c
index 04925ec..c0ee808 100644
--- a/networking/libiproute/ll_proto.c
+++ b/networking/libiproute/ll_proto.c
@@ -13,7 +13,7 @@
 #include "utils.h"
 
 #if defined(__GLIBC__) && __GLIBC__ >=2 && __GLIBC_MINOR__ >= 1
-#include <net/ethernet.h>
+#include <net/if_ether.h>
 #else
 #include <linux/if_ether.h>
 #endif
diff --git a/networking/nameif.c b/networking/nameif.c
index d02c2c1..c4a0f13 100644
--- a/networking/nameif.c
+++ b/networking/nameif.c
@@ -13,7 +13,7 @@
 #include "libbb.h"
 #include <syslog.h>
 #include <net/if.h>
-#include <netinet/ether.h>
+#include <netinet/if_ether.h>
 #include <linux/sockios.h>
 
 #ifndef IFNAMSIZ
diff --git a/networking/udhcp/dhcpc.c b/networking/udhcp/dhcpc.c
index 78aabed..3d46d6b 100644
--- a/networking/udhcp/dhcpc.c
+++ b/networking/udhcp/dhcpc.c
@@ -28,7 +28,7 @@
 #include <asm/types.h>
 #if (defined(__GLIBC__) && __GLIBC__ >= 2 && __GLIBC_MINOR__ >= 1) || defined(_NEWLIB_VERSION)
 # include <netpacket/packet.h>
-# include <net/ethernet.h>
+# include <net/if_ether.h>
 #else
 # include <linux/if_packet.h>
 # include <linux/if_ether.h>
diff --git a/networking/udhcp/files.c b/networking/udhcp/files.c
index 49bcafb..3cd0630 100644
--- a/networking/udhcp/files.c
+++ b/networking/udhcp/files.c
@@ -6,7 +6,7 @@
  *
  * Licensed under GPLv2, see file LICENSE in this source tree.
  */
-#include <netinet/ether.h>
+#include <netinet/if_ether.h>
 
 #include "common.h"
 #include "dhcpd.h"
diff --git a/networking/udhcp/packet.c b/networking/udhcp/packet.c
index 2b7528c..570cf2e 100644
--- a/networking/udhcp/packet.c
+++ b/networking/udhcp/packet.c
@@ -9,7 +9,7 @@
 #include <netinet/in.h>
 #if (defined(__GLIBC__) && __GLIBC__ >= 2 && __GLIBC_MINOR__ >= 1) || defined _NEWLIB_VERSION
 # include <netpacket/packet.h>
-# include <net/ethernet.h>
+# include <net/if_ether.h>
 #else
 # include <asm/types.h>
 # include <linux/if_packet.h>
diff --git a/networking/udhcp/socket.c b/networking/udhcp/socket.c
index 0ed7ad1..2138fa4 100644
--- a/networking/udhcp/socket.c
+++ b/networking/udhcp/socket.c
@@ -25,7 +25,7 @@
 #include <net/if.h>
 #if (defined(__GLIBC__) && __GLIBC__ >= 2 && __GLIBC_MINOR__ >= 1) || defined _NEWLIB_VERSION
 # include <netpacket/packet.h>
-# include <net/ethernet.h>
+# include <net/if_ether.h>
 #else
 # include <asm/types.h>
 # include <linux/if_packet.h>
diff --git a/networking/zcip.c b/networking/zcip.c
index 6fa265a..4cef264 100644
--- a/networking/zcip.c
+++ b/networking/zcip.c
@@ -23,8 +23,8 @@
 // - avoid silent script failures, especially under load...
 // - link status monitoring (restart on link-up; stop on link-down)
 
-#include <netinet/ether.h>
-#include <net/ethernet.h>
+#include <netinet/if_ether.h>
+#include <net/if_ether.h>
 #include <net/if.h>
 #include <net/if_arp.h>
 #include <linux/if_packet.h>
